index.html

```diff
@@ -204,55 +204,1951 @@ <h3>
+        </li>
+      </ol>
+      <p>
+        To <dfn data-lt="parsing the Report descriptor">parse the Report
+        descriptor</dfn>, run the following steps:
+      </p>
+      <ol>
+        <div class="note">
+          <p>
+            The system will typically read the [=Report descriptor=] [=byte
+            sequence=] for each [=HID interface=] when a device is first
+            connected and cache the values in the system registry. If the
+            [=Report descriptor=] is not available but the system provides
+            information that can be used to reconstruct the layout of each
+            report, the user agent may use an alternate algorithm instead of
+            parsing the [=Report descriptor=].
+          </p>
+          <p>
+            The [=Report descriptor=] format is described in <a href=
+            "https://www.usb.org/document-library/device-class-definition-hid-111">
+            Device Class Definition for HID 1.11</a> section 6.2.2. The
+            algorithm below is intended to implement the same parsing algorithm
+            described in that section.
+          </p>
+          <p>
+            The [=Report descriptor=] contains three classes of items.
+            <dfn>Main items</dfn> are used to define and group data fields.
+            <dfn>Global items</dfn> and <dfn>Local items</dfn> describe the
+            data fields. [=Local items=] define characteristics that only apply
+            to the next [=Main item=], while [=Global items=] define
+            characteristics that affect all subsequently defined [=Main
+            items=].
+          </p>
+          <p>
+            Each item has a <dfn>item data field</dfn> containing 0, 1, 2, or 4
+            bytes of data associated with the item and a <dfn>bSize field</dfn>
+            that describes how many bytes are used to store the [=item data
+            field=]. Each item also has a tag that identifies its type.
+          </p>
+        </div>
+        <li>Let |items| be the result of converting the [=Report descriptor=]
+        [=byte sequence=] into a [=list=] of [=Main items=], [=Global items=],
+        and [=Local items=].
+        </li>
+        <li>Initialize |collections:sequence&lt;HIDCollectionInfo&gt;| to be an
+        empty sequence of {{HIDCollectionInfo}}.
+        </li>
+        <li>Initialize |ancestors| to be an empty [=stack=].
+        </li>
+        <li>Initialize |localState| to be an empty [=ordered map=].
+        </li>
+        <li>Initialize |usages| to be an empty sequence of `unsigned long`.
+        </li>
+        <li>Initialize |stringIndices| to be an empty [=list=].
+        </li>
+        <li>Initialize |globalStack| to be an empty [=stack=].
+        </li>
+        <li>Initialize |initialGlobalState| to be an empty [=ordered map=].
+        </li>
+        <li>Initialize |reportId| to 0.
+        </li>
+        <li>Set |initialGlobalState|[`"unitSystem"`] to
```

Comment:
You can reference {{HIDReportItem/unitSystem}} here.
```suggestion
        <li>Set |initialGlobalState|["{{HIDReportItem/unitSystem}}"] to
```

Reply:
Done